src/client/components/ReviewPost/ReviewPost.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import profileImage from '../../assets/images/portraits/unknown-black.png';
+import './ReviewPost.styles.css';
+
+export const ReviewPost = ({ name, description, date, profileImageUrl }) => {
+  return (
+    <div className="review-section">
+      <div className="profile-image">
+        <img src={profileImageUrl || profileImage} alt="Userimage" />
+      </div>
+
+      <div className="name-date">
+        <span className="name">{name} </span>
+        <span className="date-time">{date.toLocaleString()}</span>
+      </div>
+
+      <div />
+      <span className="description">{description}</span>
```

Comment:
Everything seems good, I just notice that the description uses this color: #9A9A9A, you might need a different class name, as we have other components using that class